app/assets/stylesheets/application.scss

```diff
@@ -748,3 +748,27 @@ span.inactive {
 }
 
 .o-65 { opacity: 0.65; }
+
+.card-chip {
```

Comment:
@jamiecarter7 the CSS for these are the same for overdue patients called cards. Are these duplicates?

Reply:
They aren't - I realise that I left the css as styles in the 'patients called' card. 

This is my attempt to convert to a class. I can retro fit this into the other card.

Reply:
Oh, in that case, could you use these classes to replace the inline CSS on the called cards as well?

Reply:
yes, once this is merged to prod I can make those css class changes to the called card 👍